x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrailFilterTests.java

```diff
@@ -1554,7 +1566,7 @@ public void testIndicesFilter() throws Exception {
     }
 
     private <T> List<T> randomListFromLengthBetween(List<T> l, int min, int max) {
-        assert (min >= 0) && (min <= max) && (max <= l.size());
+        assert min >= 0 && min <= max && max <= l.size();
```

Comment:
keep the parentheses?

Reply:
I though I can spot when Eclipse does these check while validating the commit diff, but it looks like I can't. I have removed the check. I have long entertained the idea of using vim and things like this are only stoking it...